### ISSUE TITLE
strange loss for restored model

### ISSUE THREAD

**Issue by xjjs** [VisualComputingInstitute/TrackR-CNN#102]
Hi,  @pvoigtlaender

For training, I have changed the "num_epochs" in config_file to 15, and set "freeze_batchnorm" to false, and the loss is about 0.2 after 15 epochs finished. 
Then I changed the "num_epochs" to 16 and train again, the model is restored from conv3d_sep2-00000015, however, the initial loss is very large, which is about "1e6 to 1e7".

**Comment by pvoigtlaender** [comment]
Hi,

why do you set freeze_batchnorm to false? I cannot recommend that.
Especially if you use a model trained with frozen batch norm and then unfreeze batch norm, the input to the layer will be completely unexpected and then the output should be kind of random which can very well be the cause of such a high loss value.

**Comment by xjjs** [comment]
Hi, @pvoigtlaender 
Thanks for your prompt reply. 
When the freeze_batchnorm is true, the loss  falling slowly, so I set it to false. And it seems that unfreezing the batch norm may be the cause.

**Comment by xjjs** [comment]
> Hi, @pvoigtlaender
> 
> For training, I have changed the "num_epochs" in config_file to 15, and set "freeze_batchnorm" to false, and the loss is about 0.2 after 15 epochs finished.
> Then I changed the "num_epochs" to 16 and train again, the model is restored from conv3d_sep2-00000015, however, the initial loss is very large, which is about "1e6 to 1e7".

I have found the reason, if I keep the "freeze_batchnorm" false after restoring the model, then the initial loss is still small. The switch of "freeze_batchnorm" determines the final computation graph.
